### PR TITLE
Events Insights

### PR DESCRIPTION
**Pull request by hpneo**
I connected the Event Insights view with the API and added a couple of helpers.

### CODE REVIEW

File: src/components/EventDetailPage/EventDetailPage.js [hunk 1]
```diff
@@ -20,12 +22,28 @@ import { ReactComponent as MoreIcon } from "icons/more.svg";
 import styles from "./EventDetailPage.module.css";
 
 const EventDetailPage = () => {
-  const { path, url } = useRouteMatch();
-  const event = {
-    name: "Rock against the Rona",
-    published: true,
+  const { path, url, params } = useRouteMatch();
+  const [_loading, setLoading] = useState(false);
```
Comment (alessandrapereyra) [inline src/components/EventDetailPage/EventDetailPage.js:26]: Shouldn't this be true?

Comment (hpneo) [reply src/components/EventDetailPage/EventDetailPage.js:26]: right

File: src/components/EventDetailPage/EventDetailPage.js [hunk 2]
```diff
@@ -20,12 +22,28 @@ import { ReactComponent as MoreIcon } from "icons/more.svg";
...
+  const event =
+    useNestedEntity("events", params.id, {
+      eventAssignments: {},
+      metrics: {},
+    }) || {};
+
+  // eslint-disable-next-line react-hooks/exhaustive-deps
+  const loadData = async () => {
+    await fetchEvent();
+    setLoading(true);
```
Comment (alessandrapereyra) [inline src/components/EventDetailPage/EventDetailPage.js:40]: Let's set this to false.

Comment (hpneo) [reply src/components/EventDetailPage/EventDetailPage.js]: Done 😞

**Pull request merged**